disputer/disputer.js

```diff
@@ -225,10 +225,12 @@ class Disputer {
       const withdraw = this.empContract.methods.withdrawLiquidation(liquidation.id, liquidation.sponsor);
 
       // Confirm that dispute has eligible rewards to be withdrawn.
-      let withdrawAmount, gasLimit;
+      let withdrawAmount, gasEstimation;
       try {
-        withdrawAmount = revertWrapper(await withdraw.call({ from: this.account }));
```

Comment:
I don't think you want to remove the `revertWrapper`, just call `revertWraper` on the return value of `.call()`